features/util.py

```diff
@@ -52,12 +52,18 @@ class ProDebPaths:
     ubuntu_advantage_tools: str
     ubuntu_advantage_pro: str
 
-    def non_cloud_pro_image_debs(self) -> List[Tuple[str, str]]:
-        return [
-            ("ubuntu-pro-client", self.ubuntu_pro_client),
-            ("ubuntu-advantage-tools", self.ubuntu_advantage_tools),
-            ("ubuntu-pro-client-l10n", self.ubuntu_pro_client_l10n),
-        ]
+    def non_cloud_pro_image_debs(self, series: str) -> List[Tuple[str, str]]:
+        if series in ("xenial", "bionic", "focal", "jammy"):
+            return [
+                ("ubuntu-pro-client", self.ubuntu_pro_client),
+                ("ubuntu-advantage-tools", self.ubuntu_advantage_tools),
+                ("ubuntu-pro-client-l10n", self.ubuntu_pro_client_l10n),
+            ]
+        else:
+            return [
+                ("ubuntu-pro-client", self.ubuntu_pro_client),
+                ("ubuntu-pro-client-l10n", self.ubuntu_pro_client_l10n),
+            ]
 
     def cloud_pro_image_debs(self) -> List[Tuple[str, str]]:
```

Comment:
Above calls `debs.cloud_pro_image_debs(series)` so I think we need a similar modification to accept `series` down here but for `ubuntu-advantage-pro`